src/main/java/Duke.java

```diff
@@ -1,10 +1,230 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    private static List<Task> taskList = new ArrayList<>();
+
+    private static Storage storage;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
+        final String LOGO = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        final String GREETING_MESSSAGE = "Hello! I'm Duke\n"
+                + "What can I do for you?\n";
+        final String GOODBYE_MESSAGE = "Bye. Hope to see you again soon!\n";
+
+        System.out.println("Hello from\n" + LOGO);
+
+        printTextWithDivider(GREETING_MESSSAGE);
+
+        storage = new Storage();
+
+        taskList = storage.loadTasks();
+
+        Scanner sc = new Scanner(System.in);
+
+        while (sc.hasNextLine()) {
+            String command = sc.nextLine();
+
+            // Quit when user enters "bye"
+            if (command.equals("bye")) {
+                printTextWithDivider(GOODBYE_MESSAGE);
+                break;
+            }
+
+            // Execute the command
+            executeCommand(command);
+        }
+
+    }
+
+    /**
+     * Prints out the given text with dividers to the console
+     *
+     * @param text The specified text to be printed to the console
+     */
+    public static void printTextWithDivider(String text) {
+        final String divider = "-".repeat(80) + "\n";
+        System.out.println(divider + text + divider);
+    }
+
+    /**
+     * Execute the command entered by user
+     *
+     * @param command The specified command
+     */
+    private static void executeCommand(String command) {
+        // Limit the words to 2
+        String[] inputs = command.split(" ", 2);
+
+        try {
+            switch (inputs[0]) {
+                // List out all tasks
```

Comment:
I think that the switch case here isn't following the coding standard they've given to us namely the standard below with the indentation.

```java
switch {
case 1:
    // ...
case 2:
    // ...
default:
    break;
}
```